index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/4e96f49cc5.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#"><h1 class="nav-branding">My Logo</h1></a>
+        <ul class="nav-menu">
+          <li class="nav-item nav-link"><a href="#">Portfolio</a></li>
+          <li class="nav-item nav-link"><a href="#">About</a></li>
+          <li class="nav-item nav-link"><a href="#">Contact</a></li>
+        </ul>
+        <div class="hambuger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
```

Comment:
- [ OPTIONAL ] Good implementation of the hamburger but I think it's not a good practice to implement it in this way, you can use the hamburger icon given by [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Example of how you can implement it:
```
  <nav class="nav-bar">
    <div class="hamburger">
       <a href="#">
          <img src="icon-image.png">
       </a>
      </div>
  </nav>
```